src/components/ui/DataGrid/DataGrid.svelte

```diff
@@ -80,6 +80,23 @@
   let previousSelectedRowId: RowId | null = null;
 
   $: {
+    const seen_set = new Set<number>();
+    rowData.forEach(rowDatum => {
+      if (!seen_set.has(rowDatum[idKey])) {
+        // Non duplicate case
+        seen_set.add(rowDatum[idKey]);
+      } else {
+        // Found duplicate, write error message
+        console.error(
+          `%c Grid Problems? Look Here!
+A DataGrid has had multiple rows keyed over the same ID - ensure no two rows have the same value for the \`${String(
```

Comment:
I don't think you need to pass `idKey` into a `String` object since the template string will implicitly stringify it and `idKey` should already be a string.

Reply:
For some reason, a lint was failing on the possibility of implicitly casting a `Symbol` to a string at runtime which you can't do or something, and it was on this line. There isn't actually a `Symbol` at play here - `idKey` is type `string` - but this resolved the lint so..... yeah!

Reply:
Ooh! Sorry about that! This is fine then. Thank you!